### PR TITLE
adquery: added device ip and ua to bidder request

### PR DESCRIPTION
**Pull request by wgalecki**
* adquery: added device ip and ua to bidder request
* adquery: passing client ip as x-forwarded-for header
* adquery: updated adapter endpoint
* adquery: passing ipv6 to bidder, updated user sync config
* adquery: passing page url to bidder

### CODE REVIEW

**Comment by wgalecki** [comment]
Hey, sorry for the force-pushes, identified some minor issues and wanted to fix them to make the review process smoother.

File: go.mod
Comment (Sonali-More-Xandr) [inline go.mod:1]: Can you please undo changes made to go.mod?

Comment (wgalecki) [reply go.mod:1]: Yes, sorry about that, I also removed the `test` directory which I commited inadvertently.

File: static/bidder-info/adquery.yaml [hunk 1]
```diff
@@ -1,16 +1,13 @@
-endpoint: "https://bidder.adquery.io/prebid/bid"
+endpoint: https://bidder2.adquery.io/prebid/bid
```
Comment (onkarvhanumante) [inline static/bidder-info/adquery.yaml:1]:
```
% curl -i https://bidder2.adquery.io/prebid/bid
HTTP/2 204
server: nginx/1.18.0 (Ubuntu)
date: Tue, 14 Nov 2023 11:45:28 GMT
x-adq-t: r:1123;p:0;
cache-control: max-age=-172800, must-revalidate, private
pragma: no-cache
expires: Sun, 12 Nov 2023 11:45:28 GMT
access-control-allow-origin: *
access-control-allow-methods: POST, GET, OPTIONS, DELETE, PUT
access-control-max-age: 1000
access-control-allow-headers: x-requested-with, Content-Type, origin, authorization, accept, client-security-token
curl: (56) HTTP/2 stream 1 was reset
```

endpoint is reachable

**Review by onkarvhanumante — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**